docs/workflow/optimize.md

```diff
@@ -1,104 +1,192 @@
 # How to Optimize Your Shape Model?
 
-ShapeWorks uses the processed data groomed to the sign distance transform for optimization step.
-ShapeWorks uses the Particle Based Modeling (PBM) approach to constructs a correspondence 
-point model of shape, which describes shape variation by choosing a discrete set of corresponding points 
-on shape surfaces whose relative positions can be statistically analyzed. Consider a statistical sample of N surface 
-representations embeded in a d-dimensional Cartesian space(typically, d= 2, d= 3). A model for shape variation is constructed
-by choosing a set of M, d-dimensional points on each of the N surfaces. Each of the points is called a *correspondence*  point
-and Collectively, the set of M points is known as the *configuration*.
-The PBM algorithm models the correspondence positions as sets of dynamic particles that are constrained to lie on the surface of the sample set.
-The optimization is based on the idea of treating correspondence position in configuration space as a random variable, while
-simultaneously treating correspondence configuration as a random variable.
-
-Correspondence positions are optimized by gradient descent on an energy function that balances the negative entropy of 
-the distribution of particles in configuration space with the positive entropy of the distribution of the configurations
-in shape space. The method is to consider ![equation](../img/workflow/z_k.png)
- both as observations on a dM × 1 vector random
-variable Z and as N samples of M observations on N , d × 1 vector random variables ![equation](../img/workflow/x_k.png)
- . The optimization to establish correspondence minimizes the energy function
+## Particle-based Shape Modeling
+
+ShapeWorks constructs *statistically optimal anatomical mapping across different shape samples* by automatically computing a dense set of corresponding landmark positions on a set of anatomy segmentations or surface meshes and *does not rely on any specific surface parameterization*. 
```

Comment:
add geometrically consistent in this sentence basically define what is "in correspondence"

Reply:
done